rest-service/manager_rest/storage/management_models.py

```diff
@@ -141,6 +149,13 @@ class User(SQLModelBase, UserMixin):
     password = db.Column(db.String(255))
     api_token_key = db.Column(db.String(100))
 
+    tenant_associations = db.relationship(
+        'UserTenant',
+        back_populates='user',
```

Comment:
Should this be `user` or `users`?

Reply:
The back_populates value should match the attribute value defined in `management_models.UserTenant` which is called `user` because from the association object point of view this is a one-to-many relationship.

Reply:
Ah, ok. This is the secondary table. Got it.